src/gui/folder.cpp

```diff
@@ -841,6 +841,11 @@ void Folder::slotTransmissionProgress(const ProgressInfo &pi)
 // a item is completed: count the errors and forward to the ProgressDispatcher
 void Folder::slotItemCompleted(const SyncFileItemPtr &item)
 {
+    if (item->_instruction == CSYNC_INSTRUCTION_NONE || item->_instruction == CSYNC_INSTRUCTION_UPDATE_METADATA) {
+        // We only care about the updates that deserve to be shown in the UI
+        return;
+    }
```

Comment:
Including `CSYNC_INSTRUCTION_NONE` sounds wrong, would it be possible to check at the emit point?

Reply:
CSYNC_INSTRUCTION_NONE should never happen. But i'm putting it there as it goes hand to hand with UPDATE_METADATA. (Should the sync engine change, and emit the signal with CSYNC_INSTRUCTION_NONE in the future, we probably want to filter it there anyway)
I could remove it from this check if you think this is confusing.